client/src/modules/reports/generate/stock_movement_report/stock_movement_report.html

```diff
@@ -24,59 +24,64 @@ <h3 translate>REPORT.STOCK_MOVEMENT_REPORT.TITLE</h3>
 
           <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate autocomplete="off">
 
-            
-            <bh-fiscal-year-select
-              fiscal-id="ReportConfigCtrl.reportDetails.fiscal_id"
-              on-select-fiscal-callback="ReportConfigCtrl.onSelectFiscalYear(fiscalYear)">
-            </bh-fiscal-year-select>
-            
-            <bh-period-selection
-              fiscal-year-id="ReportConfigCtrl.reportDetails.fiscal_id"
-              period-id="ReportConfigCtrl.reportDetails.period_id"
-              on-select-callback="ReportConfigCtrl.onSelectPeriod(period)">
-            </bh-period-selection>
-          
-              <!-- movements flux -->
-          <bh-flux-select
-            on-change="ReportConfigCtrl.onFluxChange(flux)"
-            flux-ids="ReportConfigCtrl.reportDetails.flux_id">
-          </bh-flux-select>
+            <!-- select depot -->
+            <bh-depot-select
+              depot-uuid="ReportConfigCtrl.reportDetails.depot_uuid"
+              on-select-callback="ReportConfigCtrl.onSelectDepot(depot)"
+              required="true">
+              <bh-clear on-clear="ReportConfigCtrl.clear('depot_uuid')"></bh-clear>
+            </bh-depot-select>
 
+            <!-- date interval -->
+            <bh-date-interval
+              date-from="ReportConfigCtrl.reportDetails.dateFrom"
+              date-to="ReportConfigCtrl.reportDetails.dateTo"
+              limit-min-fiscal
+              required="true">
+            </bh-date-interval>
+          
+            <!-- movements flux -->
+            <bh-flux-select
+              on-change="ReportConfigCtrl.onFluxChange(flux)"
+              flux-ids="ReportConfigCtrl.reportDetails.flux_id">
+            </bh-flux-select>
 
-          <div class="form-group">
-            <label translate>FORM.LABELS.TYPE</label>
-                        
-            <div class="radio">
-              <label>
-                <input type="radio" name="movement_number" value="movement_number" ng-model="ReportConfigCtrl.reportDetails.reportType"/>
-                <span translate>FORM.LABELS.MOVEMENT_NUMBER</span>
-              </label>
-            </div>
-            <div class="radio">
-              <label>
-                <input type="radio" name="value" value="value" ng-model="ReportConfigCtrl.reportDetails.reportType"/>
-                <span translate>FORM.LABELS.MOVEMENT_VALUE</span>
-              </label>
+            <!-- movement or value -->
+            <div class="form-group">
+              <label translate>FORM.LABELS.TYPE</label>
+              <bh-clear on-clear="ReportConfigCtrl.clear('reportType')"></bh-clear>
+              <div class="radio">
+                <label>
+                  <input type="radio" name="movement_number" value="movement_number" ng-model="ReportConfigCtrl.reportDetails.reportType"/>
```

Comment:
Can you make these values selected by default?  When I don't select anything, I get a valid report:
![image](https://user-images.githubusercontent.com/896472/109307897-9ac8f280-7841-11eb-9f7f-4810c590c7d9.png)

So, one of these options is selected by default, but I don't know which as a user until I click it.  Can you select them?

Reply:
I would also remove the `clear()` button, since all that does is reset the default option.  It will either show one or the other, having these fields blank doesn't make sense.